cmd/crc/cmd/daemon_darwin.go

```diff
@@ -7,8 +7,11 @@ import (
 	"github.com/code-ready/crc/pkg/crc/constants"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	crcversion "github.com/code-ready/crc/pkg/crc/version"
+	"github.com/code-ready/crc/pkg/os/launchd"
 )
 
+const daemonAgentLabel = "com.github.crc.daemon"
```

Comment:
Not sure why the `const` is duplicated twice. `dev.crc.daemon` or `com.redhat.crc.daemon` would be better labels in my opinion.